src/ServicesGrid/ServiceItem/ServiceItem.scss

```diff
@@ -29,22 +26,41 @@
     color: $white;
     right: 10px;
     top: 10px;
-    background-color: rgba(0, 0, 0, 0.2);
+    background-color: darken($color3, 2);
     height: 3rem;
     width: 3rem;
     text-align: center;
     line-height: 3rem;
     border-radius: 50%;
 
-    .svg-inline--fa {
+    .icon {
       transform: scale(1.2);
+      path {
+        fill: $white;
+      }
+    }
+  }
+  .learn-more {
+    margin-top: auto;
+    margin-bottom: 0;
+    color: $white;
+    text-align: center;
+    font-weight: 700;
+
+    .icon {
+      path {
+        fill: $white;
+      }
```

Comment:
I don't think we need this as we have it further up

Reply:
It's for a different icon